libraries/chain/controller.cpp

```diff
@@ -2034,6 +2031,8 @@ struct controller_impl {
 
       fork_db.apply<void>(finish_init);
 
+      replaying = false;
```

Comment:
Should `replaying` be scoped so it will always be reset to `false` even exceptions happen above?